Dockerfile

```
@@ -5,8 +5,7 @@ WORKDIR /bin_dir
 
 RUN apk add --no-cache \
     # Builder deps
-    curl=~7 \
-    unzip=~6 && \
+    curl=~7 && \
```

```suggestion
    curl=~7 && \
    unzip=~6 && \
```

unzip needed for terraform installation